docker/Dockerfile.lumi

```diff
@@ -83,7 +88,7 @@ ENV LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
 
 # Install torch
 RUN pip install --upgrade pip
-RUN pip install torch==2.1.1 torchvision==0.16.1 torchaudio==2.1.1 --index-url https://download.pytorch.org/whl/rocm5.6
+RUN pip install --no-cache-dir torch==2.3.0 torchvision==0.18.0 torchaudio==2.3.0 --index-url https://download.pytorch.org/whl/rocm6.0
```

Comment:
Can we just go to torch 2.3.1?

Reply:
https://github.com/allenai/OLMo/pull/651/commits/e0efc20dd38810fe8f01bc8e9cc7bf682a0576bb